### PR TITLE
Fix links to GDrive after migration

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: handbook/ce/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The Customer Engineering team partners with the [Sales team] (../sales/index.md)
 	* [Sales-to-CE new customer handover process](../sales/sales_to_ce_handover.md)
 	* [CE-to-Engineering issue handover process](ce_to_eng_handover.md)
 * [QBR Guide](qbr_guide.md)
-* [Guide to demoing the Campaigns feature](https://drive.google.com/drive/u/0/folders/1ZI9A_u8K7cxQS_-WFbkG75PcqgkGpPGI) (video recording)
+* [Guide to demoing the Campaigns feature](https://drive.google.com/drive/folders/18Sa_NpsVRvVV8MIvuXyoDEinpEf8fbGn) (video recording)
```
Comment (dadlerj) [inline handbook/ce/index.md:19]: @justdueck can you please confirm this replacement folder link is correct?

File: handbook/ce/onboarding.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ NOTE: Request access/licenses to these tools in [#it-tech-ops](https://sourcegra
 - Demo Recordings
   - [101 Demo](https://drive.google.com/file/d/1VUZ0rnZQpNgjtGDI0tMC-h-OtL0Czz8H/view?usp=sharing) (dan@)
   - [Product Info Session](https://youtu.be/iTBTri_q5MA) (jonah@)
-  - [CE-to-CE Demo walkthrough](https://drive.google.com/drive/folders/1CN_9QLCPxG8-MB06zMgsvT-RjYRnEWYZ?usp=sharing) 
+  - [CE-to-CE Demo walkthrough](https://drive.google.com/drive/folders/1rf2E2KLBztH1Qe2shq3p39afF5mLK7m1)
```
Comment (dadlerj) [inline handbook/ce/onboarding.md:122]: @justdueck can you please confirm this replacement folder link is correct?

File: handbook/marketing/website_style_guide.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Sourcegraph uses the following styles on <a href="https://about.sourcegraph.com"
 
 ## Logos and icons
 
-- [All logo formats](https://drive.google.com/drive/folders/17HFjTFe4q8KLXsu7LoLS_lIgOT1OARiP?usp=sharing)
+- [All logo formats](https://drive.google.com/drive/folders/1lvSh9Q1aFsewAZ-_0tYIIH_nsKPPY0SR)
```
Comment (dadlerj) [inline handbook/marketing/website_style_guide.md:10]: @rrhyne  can you please confirm this replacement folder link is correct? I don't see anything IN the folder, which is a bit concerning. After the move, I didn't get any indication that this folder hadn't been moved correctly, so let me know if you see any issues.

File: handbook/ops/sales-ops/onboarding.md [hunk 4]
```diff
@@ -25,13 +25,13 @@ Start by reading the [operations team onboarding page](../onboarding.md). Once y
 - Read our [customer case studies](https://about.sourcegraph.com/customers)
 - Read our [customer personas](../../marketing/personas.md)
 - Review our sales content and collateral
-  - [Sales content & templates](https://drive.google.com/drive/u/0/folders/1Og99CfsVetXCutZBC01q6Cx54dX5YiyZ)
-  - [ROI calculators](https://drive.google.com/drive/u/0/folders/1dBWklUaRowyj9ZKPU7fKwgPKvsNdfhoy)
-  - [Customer order forms](https://drive.google.com/drive/u/0/folders/1VZGSm7NfcNXv_d2IZ5n6Jl1zwpMoPN7a)
+  - [Sales content & templates](https://drive.google.com/drive/folders/14guSNICPX5bHxUxXIeb1web6MN8z7FA4)
```
Comment (dadlerj) [inline handbook/ops/sales-ops/onboarding.md:28]: @joekirscherSG  can you please confirm that all of the replacement links in this folder are correct?

File: handbook/product/onboarding/index.md [hunk 5]
```diff
@@ -117,22 +117,22 @@ You'll find we have a strong base to work from, but we are in the early stages o
       - [Iconify](https://www.figma.com/community/plugin/735098390272716381/Iconify) - We use the material design icons which can be searched and included with this plugin
       - [Style organizer](https://www.figma.com/community/plugin/816627069580757929/Style-Organizer) - helps us manage color
       - [Data lab](https://www.figma.com/community/plugin/740286071386014712/Data-Lab) - populates layers with data. This helps us provide more accurate designs and avoid tedious text generation
-      - [Data for design](https://drive.google.com/drive/folders/1Xw7t1rIWRTg3cJ1_v-A40FGKCLE9m9Pg?usp=sharing)
+      - [Data for design](https://drive.google.com/drive/folders/1UPxQ4Ln_JH7KNBVGP6ZepSK5WiGWfVDO)
```
Comment (dadlerj) [inline handbook/product/onboarding/index.md:120]: @rrhyne  can you please confirm this replacement folder link is correct?

**Comment by nicksnyder** [comment]
Since a bunch of links are broken, can we optimistically merge this PR and then resolve the remaining questions async?

**Comment by dadlerj** [comment]
Yeah, merge conflicts are appearing fast. @justdueck @rrhyne please do review the changes here whenever you get the chance.

**Pull request merged**